services/brig/test/integration/API/Provider.hs

```diff
@@ -954,11 +983,13 @@ listTeamServiceProfilesByPrefix
     -> UserId
     -> TeamId
     -> Maybe Text
+    -> Bool               -- ^ Filter out disabled
     -> Int
     -> Http ResponseLBS
-listTeamServiceProfilesByPrefix brig uid tid mbPrefix size = get $ brig
+listTeamServiceProfilesByPrefix brig uid tid mbPrefix filterDisabled size = get $ brig
     . paths ["teams", toByteString' tid, "services", "whitelisted"]
     . maybe id (queryItem "prefix" . toByteString') mbPrefix
+    . (if filterDisabled then id else queryItem "filter_disabled" "false")
```

Comment:
The reason I'm not just passing `filter_disabled` all the time is that clients won't, and I'd like to kinda mimic a client here.